plan/physical_plan_builder.go

```diff
@@ -828,7 +823,7 @@ func (p *Aggregation) convert2PhysicalPlanHash() (*physicalPlanInfo, error) {
 		}
 	}
 	if !distinct {
-		if x, ok := childInfo.p.(physicalDistSQLPlan); ok {
+		if x, ok := childInfo.p.(physicalDistSQLPlan); ok && !x.isMemDB() {
```

Comment:
Hi @coocood and @hanfei1991,

I have tested this way at my local. Pushing agg to mem table doesn't run into error any more. But it looks like that the mem table is always read physcially, since the innernext of executor.go will not be called again.

besides it, what is the difference between PhyscialPlanFinalHash and PhysicalPlanCompleteHash.

regards,
Yu